repositories/case_repository.go

```diff
@@ -413,6 +413,25 @@ func (repo *MarbleDbRepository) GetCasesFileByCaseId(ctx context.Context, exec E
 	)
 }
 
+func (repo *MarbleDbRepository) GetCasesWithPivotValue(ctx context.Context, exec Executor, orgId, pivotValue string) ([]models.Case, error) {
+	if err := validateMarbleDbExecutor(exec); err != nil {
+		return nil, err
+	}
+
+	sql := NewQueryBuilder().
+		Select(columnsNames("c", dbmodels.SelectCaseColumn)...).
+		From(dbmodels.TABLE_DECISIONS + " d").
+		InnerJoin("cases c on c.id = d.case_id").
+		Where(squirrel.Eq{
+			"d.org_id":      orgId,
+			"d.pivot_value": pivotValue,
+		}).
+		OrderBy("c.created_at DESC").
```

Comment:
This is going to change depending on which cases we should prioritize showing. We talked about pulling up cases that were confirmed for example. Let's wait until we have all those status to do that.